### PR TITLE
Update go install and fix set Go env

### PR DESCRIPTION
**Pull request by vCaesar**
- Update go install and fix set Go env
- Update gopath explain

### CODE REVIEW

File: en/01.1.md [hunk 1]
```diff
@@ -76,15 +76,15 @@ A 64-bit operating system will show the following:
 
 ### Mac
 
-Go to the [download page](https://golang.org/dl/), choose `go1.4.2.darwin-386.pkg` for 32-bit systems and `go1.7.4.darwin-amd64.pkg` for 64-bit systems. Going all the way to the end by clicking "next", `~/go/bin` will be added to your system's $PATH after you finish the installation. Now open the terminal and type `go`. You should see the same output shown in figure 1.1.
+Go to the [download page](https://golang.org/dl/), choose `go1.4.2.darwin-386.pkg` for 32-bit systems and `go1.8.darwin-amd64.pkg` for 64-bit systems. Going all the way to the end by clicking "next", `~/go/bin` will be added to your system's $PATH after you finish the installation. Now open the terminal and type `go`. You should see the same output shown in figure 1.1.
```
Comment (appleboy) [inline en/01.1.md]: 1.4.2?

File: en/01.1.md [hunk 2]
```diff
@@ -112,11 +112,14 @@ Ubuntu is the most popular desktop release version of Linux. It uses `apt-get` t
...
-export PATH=PATH:GOROOT/bin 
-export GOPATH=HOME/gowork 
+wget https://storage.googleapis.com/golang/go1.8.linux-amd64.tar.gz
+sudo tar -xzf go1.8.linux-amd64.tar.gz -C /usr/local 
+
+# Go environment
+export GOROOT=/usr/local/go
+export GOBIN=$GOROOT/bin
+export PATH=$PATH:$GOBIN
+export GOPATH=HOME/gopath
```
Comment (appleboy) [inline en/01.1.md]: `GOPATH` default value is `$HOME/go` in go 1.8 version.

File: en/01.1.md (hunk 1 above)
Comment (vCaesar) [inline en/01.1.md]: The latest version has no 32-bit download.

File: en/01.1.md (hunk 2 above)
Comment (vCaesar) [inline en/01.1.md]: Thanks for your reminding.

**Pull request merged**